### PR TITLE
adds artifactoryKey init and cleanup scripts

### PR DESCRIPTION
**Pull request by Riturajcse**
https://github.com/Shippable/execTemplates/issues/21
https://github.com/Shippable/execTemplates/issues/22

image

- runSh jobs used to test

```
  - name: myJfrogRunShJob
    type: runSh
    on_start:
      - NOTIFY: slackNotification
    on_success:
      - NOTIFY: slackNotification
    steps:
      - IN: jfrog-appfile
      - IN: jFrog-cliconfig
      #- IN:tr1
      - TASK:
        - script: echo 'start runSh hello'
        - script: jfrog rt dl test-repository-local/package.json
        - script: cp package.json newPackage.$BUILD_NUMBER.json
        - script: ls -al
        - script: jfrog rt u newPackage.$BUILD_NUMBER.json test-repository-local
```

### CODE REVIEW

**Comment by shrivara** [comment]
Please fix the linting errors :)

File: integrations/artifactoryKey/cli/init.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  fi
+
+  _log_success "Successfully checked params"
+}
+
+init_scope_configure() {
+  _log_msg "Initializing scope configure"
+
+  jfrog_version=$(jfrog --version | awk '{print $3}' )
+  jfrog_MajorVersion=$(echo $jfrog_version | awk -F '.' '{print $1}' )
```
Comment (shrivara) [inline integrations/artifactoryKey/cli/init.sh]: Don't mix cases. This should all use underscore.

File: integrations/artifactoryKey/cli/init.sh [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  jfrog_MinorVersion=$(echo $jfrog_version | awk -F '.' '{print $2}' )
+
+  newJFrogCommand="jfrog rt config default-server --url $JFROG_URL \
+  --password $JFROG_PASSWORD --user $JFROG_USERNAME \
```
Comment (shrivara) [inline integrations/artifactoryKey/cli/init.sh]: Add 2 spaces.

File: integrations/artifactoryKey/cli/init.sh [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  --interactive=false && jfrog rt use default-server"
+
+  oldJFrogCommand="jfrog rt config --url $JFROG_URL --password \
+  $JFROG_PASSWORD --user $JFROG_USERNAME"
+
+  if [ "$jfrog_MajorVersion" -gt "1" ] ||
```
Comment (shrivara) [inline integrations/artifactoryKey/cli/init.sh]: Let's move this to a helper function to keep this method clean. See `_helpers` for a similar check for Docker.

File: integrations/artifactoryKey/cli/cleanup.sh [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  Usage:
+    $SCRIPT_NAME <resource_name> [scope]
+  "
+}
+
+cleanup_scope_configure() {
+  _log_msg "Cleaning up scope configure"
+
+  local jfrog_config_path=~/.jfrog
+  jfrog_config_path=~/.jfrog
```
Comment (shrivara) [inline integrations/artifactoryKey/cli/cleanup.sh]: Why define this twice?

**Review by shrivara — CHANGES_REQUESTED** [review]

File: integrations/artifactoryKey/cli/init.sh [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    exit 1
+  fi
+
+  _log_success "Successfully checked params"
+}
+
+init_scope_configure() {
+  _log_msg "Initializing scope configure"
+
+  newJFrogCommand="jfrog rt config default-server --url $JFROG_URL \
```
Comment (scriptnull) [inline integrations/artifactoryKey/cli/init.sh]: `new_jfrog_command`

**Review by scriptnull — APPROVED** [review]

**Pull request merged**